pkg/parcacol/parcacol.go

```diff
@@ -97,10 +96,10 @@ func flatProfileToBuffer(logger log.Logger, ls labels.Labels, schema *dynparquet
 	return buf, nil
 }
 
-func extractLocationIDs(locs []*metastore.Location) []uuid.UUID {
-	res := make([]uuid.UUID, 0, len(locs))
+func extractLocationIDs(locs []*metastore.Location) []byte {
+	b := make([]byte, 0, len(locs)*16) // UUID are 16 bytes thus multiply by 16
 	for i := len(locs) - 1; i >= 0; i-- {
-		res = append(res, locs[i].ID)
+		b = append(b, locs[i].ID[:]...)
```

Comment:
Does the go compiler optimize this to `copy(..., ...)` under the hood? If not we should probably do that here.

Reply:
Oh, yes. Good point. Will update! :+1: